crates/oxc_linter/src/snapshots/prefer_query_selector.snap

```diff
@@ -0,0 +1,143 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: prefer_query_selector
+---
+  ☞ eslint-plugin-unicorn(prefer-query-selector): Prefer `.querySelector()` over `.getElementById()`.
+   ╭─[prefer_query_selector.tsx:1:1]
+ 1 │ document.getElementById("foo");
+   · ──────────────────────────────
```

Comment:
`getElementById` is the only part that the error message targets, shall we make the span only point to `getElementById`?